### DISCUSSION TITLE
When using with Azure Function App getting Error: Cannot find module '.prisma/client'

### DISCUSSION THREAD

**Discussion by garyburtonjisc** [prisma/prisma#10059] [Q&A] [answered]
We have successfully deployed Prisma in an azure function before but this one just won't work. Its on runtime 3 with node 14. Prisma is latest version. We are using azure pipeline to build the function and zip it up and then terraform creates the infrastructure - basically uploads the zip to azure storage and then set `WEBSITE_RUN_FROM_PACKAGE` to point to that zip. Prisma client is definitely getting generated so there is no reason it should not be there.

Anyone any ideas?

**Comment by garyburtonjisc** [comment, marked as answer]
This was because my zip script wasn't including directories beginning with `.`

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
